frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -18,10 +26,179 @@ function ReaderCoptListener:onReadSettings(config)
+    self.ui.document._document:setIntProperty("window.status.pos.page.number", self.page_number)
+    self.ui.document._document:setIntProperty("window.status.pos.page.count", self.page_count)
+    self.ui.document._document:setIntProperty("crengine.page.header.chapter.marks", self.chapter_marks)
+    self.ui.document._document:setIntProperty("window.status.battery", self.battery)
+    self.ui.document._document:setIntProperty("window.status.battery.percent", self.battery_percent)
+    self.ui.document._document:setIntProperty("window.status.pos.percent", self.reading_percent)
 end
 
 function ReaderCoptListener:onSetFontSize(font_size)
     self.document.configurable.font_size = font_size
 end
 
+function ReaderCoptListener:setAndSave(property, setting, value)
+    self.ui.document._document:setIntProperty(property, value)
+    G_reader_settings:saveSetting(setting, value)
+    UIManager:broadcastEvent(Event:new("SetStatusLine", self.document.configurable.status_line, true))
+end
+
+local about_text = _([[
+Here you can set the items shown in the top status bar.
+
+The settings here will only affect CRE documents in page mode.
+
+The top status bar (per document or by default) has to be enabled in the bottom menu.]])
+
+function ReaderCoptListener:getAltStatusBarMenu()
+    return {
+        text = _("Alt status bar"),
+        sorting_hint = "status_bar",
+        sorting_hint_top = true,
+        separator = true,
+        sub_item_table = {
+            {
+                text = _("About alternate status bar"),
+                keep_menu_open = true,
+                callback = function()
+                    UIManager:show(InfoMessage:new{
+                        text = about_text,
+                    })
+                end,
+                separator = true,
+            },
+            {
+                text = _("Title"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.title == 1
+                end,
+                callback = function()
+                    self.title = 1 - self.title
+                    self:setAndSave("window.status.title", "cre_header_title", self.title)
+                end,
+            },
+            {
+                text = _("Clock"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.clock == 1
+                end,
+                callback = function()
+                    self.clock = 1 - self.clock
+                    self:setAndSave("window.status.clock", "cre_header_clock", self.clock)
+                end,
+            },
+            {
+                text = _("Page number"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.page_number == 1
+                end,
+                callback = function()
+                    self.page_number = 1 - self.page_number
```

Comment:
I would have let it pass for the previous ones :) but seeing this one (_we're adding, no, substracting 1 to the page number?!_ :), it feels a more explicite:
`self.page_number = self.page_number == 0 and 1 or 0`
is clearer than your arithmetic to toggle between 0 and 1 :)
(if doing it for this one, might as well do it for the others.)